Apps/US/DynamicsGPMigrationVendor1099/app/src/Codeunits/GPPopulateVendor1099Data.Codeunit.al

```diff
@@ -0,0 +1,430 @@
+codeunit 42003 "GP Populate Vendor 1099 Data"
+{
+    var
+        VendorTaxBatchNameTxt: Label 'GPVENDTAX', Locked = true;
+        VendorTaxNoSeriesTxt: Label 'VENDTAX', Locked = true;
+        SourceCodeTxt: Label 'GENJNL', Locked = true;
+        NoSeriesDescriptionTxt: Label 'GP Vendor 1099', Locked = true;
+        DefaultPayablesAccountCode: Code[20];
+        PostingDate: Date;
+
+    [EventSubscriber(ObjectType::Codeunit, CodeUnit::"Data Migration Mgt.", 'OnAfterMigrationFinished', '', false, false)]
+    local procedure OnAfterMigrationFinishedSubscriber(var DataMigrationStatus: Record "Data Migration Status"; WasAborted: Boolean; StartTime: DateTime; Retry: Boolean)
```

Comment:
We agreed on the new codeunit for moving US specific data. This event should be there and bind this codeunit and potentially other codeunits.